readme.md

```diff
@@ -202,6 +203,21 @@ got.stream('github.com')
 
 `redirect` event to get the response object of a redirect. The second argument is options for the next request to the redirect location.
 
+##### .on('uploadProgress', progress)
+##### .on('downloadProgress', progress)
+
+Progress events for uploading (sending request) and downloading (receiving response). The `progress` argument is an object like:
+
+```js
+{
+	percent: 0.24,
```

Comment:
`1024` bytes send against `10240` in total means 10% and thus `percent` should be `0.1`.